docs/02-iaas/mandatory-and-supported-os-services.md

```diff
@@ -0,0 +1,55 @@
+# SCS mandatory and supported OpenStack services
+
+There are many OpenStack APIs and their services that can be applied on IaaS-Level.
+These services have differences in the quality of their implementation and liveness and some of them may be easily omitted when creating an IaaS-Deployment.
+To fulfill all SCS-provided standards there are only some of these APIs required.
+More but not all OpenStack services are tested or integrated in the reference implementation.
+This document will give readers insight about how the SCS looks at all the OpenStack services.
+If a cloud provides all mandatory and maybe some supported OpenStack APIs and implementation of their services it can be tested for SCS-compliance.
+Any unsupported services will not be tested.
+
+## Mandatory OpenStack services
+
+The following OpenStack services MUST be present in SCS-compliant IaaS-Deployments:
+
+| OpenStack Service | description |
+|-----|-----|
+| **Cinder** | Block Storage service |
+| **Glance** | Image service |
+| **Keystone** | Identity service |
+| **Neutron** | Networking service |
+| **Nova** | Compute service |
+| **Octavia** | Load-balancer service |
+| **Placement** | Hardware Describing Service for Nova |
+| **S3 API object storage** | No formal standard exists, many implementations: Swift, RadosGW, minio... |
+
+## Supported OpenStack services
+
+The following services MAY be present in SCS-compliant IaaS-Deployment and their integration had been tested by the SCS:
+
+| OpenStack Service | description | comment |
+|-----|-----|-------|
+| **Barbican** | Key Manager service | maybe a future mandatory service |
+| **Cloudkitty** | Rating/Billing service | maybe move to unsupported |
+| **Ceilometer** | Telemetry service | |
+| **Designate** | DNS service | |
+| **Gnocchi** | Time Series Database service | needed for Ceilometer |
+| **Heat** | Orchestration service | **maybe unsupported?** |
+| **Horizon** | Dashboard | |
+| **Ironic** | Bare Metal libraries | |
```

Comment:
Ironic is not only a library but also provides actual services, imho this fits it better (stolen from the ironic docs)?
```suggestion
| **Ironic** | Bare Metal provisioning service | |
```